<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-11-17T16:21:21 IST" name="Default suite" finished-at="2021-11-17T16:21:37 IST" duration-ms="16694">
    <groups>
    </groups>
    <test started-at="2021-11-17T16:21:21 IST" name="Default test" finished-at="2021-11-17T16:21:37 IST" duration-ms="16694">
      <class name="com.demo.automation.loginassessment.Demotest.DemoTestLogin">
        <test-method is-config="true" signature="setupClass()[pri:0, instance:com.demo.automation.loginassessment.Demotest.DemoTestLogin@4efc180e]" started-at="2021-11-17T16:21:21 IST" name="setupClass" finished-at="2021-11-17T16:21:24 IST" duration-ms="3747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupClass -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:com.demo.automation.loginassessment.Demotest.DemoTestLogin@4efc180e]" started-at="2021-11-17T16:21:24 IST" name="setupTest" finished-at="2021-11-17T16:21:32 IST" duration-ms="7383" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="test()[pri:0, instance:com.demo.automation.loginassessment.Demotest.DemoTestLogin@4efc180e]" started-at="2021-11-17T16:21:32 IST" name="test" finished-at="2021-11-17T16:21:37 IST" duration-ms="4881" status="FAIL">
          <exception class="com.demo.automation.loginassessment.Utils.MyLoginException">
            <message>
              <![CDATA[Please enter a valid input]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.demo.automation.loginassessment.Utils.MyLoginException: Please enter a valid input
at com.demo.automation.loginassessment.Utils.CommonMethods.verifyUsername(CommonMethods.java:68)
at com.demo.automation.loginassessment.Pages.LoginPage.enterUsername(LoginPage.java:22)
at com.demo.automation.loginassessment.Demotest.DemoTestLogin.test(DemoTestLogin.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.demo.automation.loginassessment.Utils.MyLoginException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.demo.automation.loginassessment.Demotest.DemoTestLogin@4efc180e]" started-at="2021-11-17T16:21:37 IST" name="teardown" finished-at="2021-11-17T16:21:37 IST" duration-ms="486" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.demo.automation.loginassessment.Demotest.DemoTestLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
